docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/install-and-configure-stripe-prerequisites.md

```diff
@@ -217,6 +217,100 @@ use Spryker\Zed\OauthClient\Communication\Plugin\Payment\AccessTokenPaymentAutho
+
+```xml
+<?xml version="1.0"?>
+<config>
+    ...
+
+    <merchant-portal-payment-settings>
+        <label>Payment Settings</label>
+        <title>Payment Settings</title>
+        <icon>payment</icon>
+        <bundle>merchant-app-merchant-portal-gui</bundle>
+        <controller>payment-settings</controller>
+        <action>index</action>
+        <pages>
+            <onboarding>
+                <label>Onboarding</label>
+                <title>Onboarding</title>
+                <icon>payment</icon>
+                <bundle>merchant-app-merchant-portal-gui</bundle>
+                <controller>payment-settings</controller>
+                <action>onboarding</action>
+                <visible>0</visible>
+            </onboarding>
+        </pages>
+    </merchant-portal-payment-settings>
+</config>
+```
+
+### Add plugins
+
+##### KernelApp
+- `\Spryker\Zed\MerchantApp\Communication\Plugin\KernelApp\MerchantAppRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
+- `\Spryker\Zed\OauthClient\Communication\Plugin\KernelApp\OAuthRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
+
+##### MessageBroker
+- `\Spryker\Zed\MerchantApp\Communication\Plugin\MessageBroker\MerchantAppOnboardingMessageHandlerPlugin` - Add this one to `\Pyz\Zed\MessageBroker\MessageBrokerDependencyProvider::getMessageHandlerPlugins()`
+- `\Spryker\Zed\KernelApp\Communication\Plugin\MessageBroker\AppConfigMessageHandlerPlugin` - Add this one to `\Pyz\Zed\MessageBroker\MessageBrokerDependencyProvider::getMessageHandlerPlugins()` (if not already present)
+
+##### AclMerchantPortal
+- `\Spryker\Zed\MerchantAppMerchantPortalGui\Communication\Plugin\AclMerchantPortal\MerchantAppMerchantPortalGuiMerchantAclRuleExpanderPlugin` - Add this one to `\Pyz\Zed\AclMerchantPortal\AclMerchantPortalDependencyProvider::getMerchantAclRuleExpanderPlugins()`
+- `\Spryker\Zed\MerchantAppMerchantPortalGui\Communication\Plugin\AclMerchantPortal\MerchantAppAclEntityConfigurationExpanderPlugin`- Add this one to `\Pyz\Zed\AclMerchantPortal\AclMerchantPortalDependencyProvider::getAclEntityConfigurationExpanderPlugins()`
+  `
+### Configure OMS
+
+1. Add the following commands to `\Pyz\Zed\Oms\OmsDependencyProvider::extendCommandPlugins()`:
+```php
+$commandCollection->add(new MerchantPayoutCommandByOrderPlugin(), 'SalesPaymentMerchant/Payout');
+$commandCollection->add(new MerchantPayoutReverseCommandByOrderPlugin(), 'SalesPaymentMerchant/ReversePayout');
+```
+
+2. Add the following conditions to `\Pyz\Zed\Oms\OmsDependencyProvider::extendConditionPlugins()`:
+```php
+$conditionCollection->add(new IsMerchantPaidOutConditionPlugin(), 'SalesPaymentMerchant/IsMerchantPaidOut');
+$conditionCollection->add(new IsMerchantPayoutReversedConditionPlugin(), 'SalesPaymentMerchant/IsMerchantPayoutReversed');
+```
+
+#### Configuration
```

Comment:
Update this title so it describes the kind of configuration added in this section